packages/client/src/runtime/utils/CallSite.ts

```diff
@@ -32,18 +33,29 @@ class EnabledCallSite implements CallSite {
     // TODO: more resilient logic to check that it's not relative to cwd
     const frame = stackFrames.find((t) => {
       // Here we are trying to find the location in the user's code which caused the error
+      if (!t.file) {
+        return false
+      }
+
+      // convert windows path to posix path
+      const posixFile = t.file.split(path.sep).join('/')
```

Comment:
Does this explain why we had weird stack traces on windows only with swc?

Reply:
No, they are couple of lines off there, and we did not filter based on path separator before this PR.